README.md

```diff
@@ -916,6 +919,62 @@ Filter characters can be deleted partially or entirely via, respectively, Backsp
 
 If the user changes or deletes a filter, the choices previously selected remain selected.
 
+#### 2.6.2.8 `:confirm_keys`
+
+You can configure which key(s) confirm your selection (`:space` and `:return` by default):
+
+```ruby
+choices = %w(Scorpion Kano Jax)
+prompt.select("Choose your destiny?", choices, confirm_keys: [:ctrl_s, ","])
+# =>
+# Choose your destiny? (Press ↑/↓ to move and Ctrl+S or , to select)
+# ‣ Scorpion
+#   Kano
+#   Jax
+```
+
+This is particularly useful in conjunction with `:filter`, as you may have choices that include spaces.
+
+```ruby
+choices = ["Jax Sr", "Jax", "Jax Jr"]
+prompt.select("Choose your destiny?", choices, confirm_keys: [:ctrl_s], filter: true)
+# =>
+# Choose your destiny? (Press ↑/↓ to move, Ctrl+S to select and letters to filter)
+# ‣ Jax Sr
+#   Jax
+#   Jax Jr
+```
+
+After the user types "Jax":
+
+```ruby
+# =>
+# Choose your destiny? (Filter: "Jax")
+# ‣ Jax Sr
+#   Jax
+#   Jax Jr
+```
+
+After the user presses space:
+```ruby
+# =>
+# Choose your destiny? (Filter: "Jax ")
+# ‣ Jax Sr
+#   Jax Jr
+```
+
+You may also pass your own key labels to be displayed in the hint:
+
+```ruby
+choices = ["Jax Sr", "Jax", "Jax Jr"]
+prompt.select("Choose your destiny?", choices, confirm_keys: [:enter, {escape: "ESC"}, {"," => "Comma (,"}])
```

Comment:
Super nit-picky - `)` missing in the `"Comma (,"` - sorry!

Reply:
oops  🤦‍♀️  fixed!